docs/src/modules/components/DemoToolbar.js

```diff
@@ -363,6 +364,71 @@ export default function DemoToolbar(props) {
     isFocusableControl,
   });
 
+  const devMenuItems = [];
+  if (process.env.STAGING === true) {
+    /* eslint-disable material-ui/no-hardcoded-labels -- staging only */
+    // eslint-disable-next-line react-hooks/rules-of-hooks -- process.env.STAGING never changes
+    const router = useRouter();
+
+    const defaultReviewID = process.env.GIT_REVIEW_ID ?? '20000';
```

Comment:
Should we exit the whole branch instead of defaulting to 20000 if GIT_REVIEW_ID is missing?

Reply:
It's a template to get a link to another PR. 20000 is just an arbitrary number to make the PR number apparent in the created link.